src/robot/model/filter.py

```diff
@@ -72,6 +77,8 @@ def start_suite(self, suite):
             suite.tests = self._filter(suite, self._included_by_tags)
         if self.exclude_tags:
             suite.tests = self._filter(suite, self._not_excluded_by_tags)
+        if self.skip_tags:
+            suite.tests = self._filter(suite, self._not_skipped_by_tags)
```

Comment:
Doesn't this mean that  `--skip` works exactly like `--exclude`? Shouldn't it rather somehow mark tests to be skipped so that they would be seen in statistics?